config/runtime.exs

```diff
@@ -78,5 +85,5 @@ case System.get_env("SERVER_API_TOKEN") do
 end
 
 if prod? do
-  config :jellyfish, JellyfishWeb.Endpoint, url: [host: host, port: 443, scheme: "https"]
+  config :jellyfish, JellyfishWeb.Endpoint, url: [scheme: "https"]
```

Comment:
Could we move all setting up configs about `JellyfishWeb.Endpoint` closes together and the second group would configs under key `:jellyfish`? By this I mean moving this if after `check_origin` if and moving `JF_SERVER_API_TOKEN` before `config :jellyfish, JellyfishWeb.Endpoint`.